android-auto-app/build.gradle.kts

```diff
@@ -17,8 +17,10 @@ android {
     versionName = "0.1.0"
     multiDexEnabled = true
     testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
-    testInstrumentationRunnerArguments = mapOf(
-      "clearPackageData" to "true"
+    testInstrumentationRunnerArguments(
+      mapOf(
```

Comment:
does it means that you should bump GP version?

Reply:
Still trying to figure out why we need to change these, seems the new gradle plugin depends on AGP 4.1 and the current AGP version in the mapbox-maps-android is 4.0.

Probably we should bump the AGP version in this repo as well.

Reply:
it will be nice :slightly_smiling_face:
mutation tests will be merged

Reply:
We will need to bump AGP version from upstream first, for now let's keep it as is.